themes/grav/templates/forms/fields/editor/editor.html.twig

```diff
@@ -5,8 +5,10 @@
 
 {% block label %}
     {% if field.label %}
-        {% set hint = field.help ? 'data-hint="' ~ field.help|tu|raw ~ '"': '' %}
```

Comment:
`|e('html_attr')` doesn't fix this particular Twig weirdness, I don't think - if that's what you meant

If you just meant my code, though, then yeah you're right - my bad over-eagerly copy-pasting